Source/Fuse.Reactive.JavaScript/FuseJS/TreeObservable.js

```diff
@@ -0,0 +1,66 @@
+
+
+function TreeObservable(state)
+{
+	if (!(this instanceof TreeObservable)) { return new TreeObservable(state); }
+
+	var store = this;
+
+	var api = 
+	{
+		get : function() {
```

Comment:
We've go mixed brace styles throughout. I'd prefer the egyptian style on JS files -- that's what Observable mainly moved towards (still mixed there though).